angularjs-portal-home/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -166,6 +166,14 @@ define(['angular', 'jquery'], function(angular, $) {
                 }
             };
 
+            this.renderURL = function renderURL(portlet) {
```

Comment:
Its a step. Someday the marketplace and layout will be the same object, someday...